.travis.yml

```diff
@@ -28,97 +28,98 @@ jobs:
   fast_finish: true
   include:
   - stage: "Main Squeak and Pharo builds"
-    env: ARCH="linux32x86" FLAVOR="squeak.cog.spur"
-  - env: ARCH="macos32x86" FLAVOR="squeak.cog.spur"
+    env: ARCH="linux32x86"    FLAVOR="squeak.cog.spur"          TESTIMAGE="Squeak-5.1"
```

Comment:
Not that https://github.com/hpi-swa/smalltalkCI/issues/407 is fixed, please update all image names accordingly (`Squeak32-5.1`, ...).